base/src/main/java/io/spine/validate/ByteStringFieldValidator.java

```diff
@@ -44,14 +47,13 @@ protected boolean isNotSet(ByteString value) {
     }
 
     /**
-     * {@inheritDoc}
+     * @inheritDoc
      *
-     * <p>Performs no action since no special options are declared for the {@link ByteString}
-     * validation.
+     * <p>All of the additional validation logic is passed as the third
+     * {@linkplain FieldValidator(FieldValue, boolean, Set )} super constructor} parameter.
```

Comment:
This line is the reason for the "unused" import reported by Codacy. Please do not add imports just the Javadoc.